client/transactions/uncaptured/test/__snapshots__/index.tsx.snap

```diff
@@ -243,77 +243,167 @@ exports[`Authorizations list renders correctly 1`] = `
                     Sort by Amount in descending order
                   </span>
                 </th>
+                <th
+                  class="woocommerce-table__header"
+                  role="columnheader"
+                  scope="col"
+                >
+                  <span
+                    aria-hidden="true"
+                  >
+                    Action
+                  </span>
+                  <span
+                    class="screen-reader-text"
+                  >
+                    Action
+                  </span>
+                </th>
               </tr>
               <tr>
                 <th
                   class="woocommerce-table__item date-time is-left-aligned is-sorted"
                   scope="row"
                 >
-                  2020-01-02 17:46:02
+                  <a
+                    class="woocommerce-table__clickable-cell"
+                    data-link-type="wc-admin"
+                    href="admin.php?page=wc-admin&path=%2Fpayments%2Ftransactions%2Fdetails&id=pi_4242"
+                    tabindex="-1"
+                  >
+                    Jan 2, 2020 / 12:46PM
+                  </a>
                 </th>
                 <td
                   class="woocommerce-table__item date-time is-left-aligned"
                 >
-                  2020-01-09 17:46:02
+                  <a
+                    class="woocommerce-table__clickable-cell"
+                    data-link-type="wc-admin"
+                    href="admin.php?page=wc-admin&path=%2Fpayments%2Ftransactions%2Fdetails&id=pi_4242"
+                    tabindex="-1"
+                  >
+                    Jan 9, 2020 / 12:46PM
+                  </a>
                 </td>
                 <td
                   class="woocommerce-table__item"
                 >
                   <a
-                    href="https://doggo.com"
+                    data-link-type="wp-admin"
+                    href="post.php?post=24&action=edit"
                   >
                     #24 from Good boy
                   </a>
                 </td>
                 <td
                   class="woocommerce-table__item is-left-aligned"
                 >
-                  <span
-                    style="color: crimson;"
+                  <a
+                    class="woocommerce-table__clickable-cell"
+                    data-link-type="wc-admin"
+                    href="admin.php?page=wc-admin&path=%2Fpayments%2Ftransactions%2Fdetails&id=pi_4242"
+                    tabindex="-1"
                   >
-                    High
-                  </span>
+                    <span>
+                      N/A
```

Comment:
This is related to my comment about `risk_level` being an integer. Switching it to an integer will correctly fix this issue (showing a correct risk level -`Normal / High / Elevated` - instead of `N/A`)

Reply:
Indeed, thanks! [2028ea7](https://github.com/Automattic/woocommerce-payments/pull/4961/commits/2028ea7057dfd02686fe73ab2f3839e2914a80f0)